finish/openLibertyCafe/src/main/java/io/openliberty/guides/openlibertycafe/OpenLibertyCafeServingWindowResource.java

```diff
@@ -43,13 +49,21 @@ public Response getReady2Serve(){
     @POST
     @Path("/{orderId}")
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "completeReadyOrder",
+               summary = "Complete a ready order",
+               description = "This operation completes an order " +
+                   "by the ServingWindow service.")
     @Tag(name = "Serving Window")
     public Response serveOrder(@PathParam("orderId") String orderId) {
         return servingWindowClient.serveOrder(orderId);
     }
 
     @DELETE
     @Produces(MediaType.APPLICATION_JSON)
+    @Operation(operationId = "resetServingWindow",
+               summary = "Clear ready orders",
+               description = "This operation remove all orders " + 
+                   "in the ServingWindow service.")
```

Comment:
Update the description to
"This operation removes all orders in the ServingWindow service."

Reply:
updated